scripts/generate_dump

```diff
@@ -1252,7 +1276,6 @@ main() {
     # Save reboot cause information
     save_cmd "show reboot-cause" reboot.cause
 
-    local asic="$(/usr/local/bin/sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)"
```

Comment:
please revert

Reply:
I made this invoking more early therefore in this place its need to remove.